modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/upgrade/registry/NotificationUpgradeStepRegistrator.java

```diff
@@ -156,6 +156,11 @@ protected String[][] getTableAndPrimaryKeyColumnNames() {
 			new DeleteStaleNotificationQueueEntriesAndNotificationTemplatesUpgradeProcess(
 				_classNameLocalService, _groupLocalService,
 				_portletFileRepository, _resourcePermissionLocalService));
+
+		registry.register(
+			"3.10.4", "3.10.5",
```

Comment:
This needs to be a major change. Please, see this [documentation](https://liferay.atlassian.net/wiki/spaces/ENGCOREUPGRADE/pages/1654719711/Upgrade+Development+Training#Schema-version:~:text=Change%20a%20column%20type.%C2%A0This%20includes%20converting%20a%20string%20field%20to%20CLOB%20(when%20it%20is%20not%20a%20Micro%20change)).